docs/src/manifolds/group.md

```diff
@@ -43,6 +43,56 @@ Pages = ["groups/GroupManifold.jl"]
 Order = [:constant, :type, :function]
 ```
 
+### Default Storage of Tangent Vectors
+
+In most groups, the storage of a tangent vector
```

Comment:
I would maybe prefer to speak about representation instead of storage.

Storage for me sounds like an _extra_ memory allocated for something (maybe some `X.p` such that `X` knows its base point); you do not do that, the text just reads a bit like that.

Reply:
And I think representation is not much my problem, we do a very similar “triick” on SPDs – the point why we need the action is maybe something I would like to understand a bit more (and since I want to understand it, documenting it somewhere would be great).

Reply:
Which action are you talking about here? The adjoint action? 

The piece of documentation I wrote explains exactly why the adjoint action is needed to compute the derivatives of the group operations when using left invariant ~storage~ representation of tangent vectors.

What else would you need in the docs?

Reply:
Ah, I did not read that careful enough, yes that should suffice for now. Also thanks for using representation, that is what we usually use for that as well when talking about different representations of points or tangent vectors.